shimmingtoolbox/masking/mask_mrs.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Creating MRS mask API
+"""
+import os
+import numpy as np
+import numpy.linalg as npl
+import nibabel as nib
+import logging
+import json
+import subprocess
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def mask_mrs(fname_input, raw_data, X, Y, Z, V):
+    """
+    Create a mask to shim single voxel MRS
+
+    Args:
+        fname_input: Input path of the fieldmap to be shimmed (both nii.gz and json file should be in this path)
```

Comment:
Add variable types in the docstring

```suggestion
        fname_input (str): Input path of the fieldmap to be shimmed (both nii.gz and json file should be in this path)
```